README.md

```diff
@@ -12,3 +12,29 @@ $ npm run storybook
 # Compiles and minifies for production
 $ npm run build-storybook
 ```
+
+## Style root color components
+
+```css
+:root {
+  --primary: #491e8c;
+  --secondary: #3b1873;
+  --dark-purple: #210e41;
```

Comment:
Maybe we could rename the purple in all these variable
```
--primary-dark
--primary-very-dark
--primary-light
--primary-very-light
```

Reply:
Okay, I will change it.

Reply:
Fixed.